server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -12,21 +13,33 @@
     PullRequestReview, PushCommit, Release, User
 
 
+async def mine_contributors(repos: Collection[str],
+                            db: databases.Database,
+                            time_from: Optional[datetime] = None,
+                            time_to: Optional[datetime] = None,
+                            with_stats: Optional[bool] = True,
+                            cache: Optional[aiomcache.Client] = None) -> List[dict]:
+    """Discover developers who made any important action in the given repositories and \
+    in the given time frame."""
+    time_from = time_from or datetime(1970, 1, 1, tzinfo=timezone.utc)
+    time_to = time_to or datetime.now(timezone.utc)
+
+    return await _mine_contributors(repos, time_from, time_to, db, with_stats, cache)
+
+
 @cached(
     exptime=5 * 60,
     serialize=marshal.dumps,
     deserialize=marshal.loads,
     key=lambda repos, time_from, time_to, **_: (
         ",".join(repos), time_from.timestamp(), time_to.timestamp()),
```

Comment:
```suggestion
    key=lambda repos, time_from, time_to, with_stats, **_: (
        ",".join(repos), time_from.timestamp(), time_to.timestamp(), with_stats),
```
sorry, missed this another important one

Reply:
woo this is a very big overlook from my side 😕, I added a simple test as well